VotingApplication/VotingApplication.Web.Api.Tests/E2E/BasicPollTests.cs

```diff
@@ -0,0 +1,603 @@
+﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Chrome;
+using Protractor;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using VotingApplication.Data.Context;
+using VotingApplication.Data.Model;
+using VotingApplication.Web.Api.Tests.E2E.Helpers;
+using VotingApplication.Web.Api.Tests.E2E.Helpers.Clearers;
+
+namespace VotingApplication.Web.Tests.E2E
+{
+    public class BasicPollTests
+    {
+        private static readonly string ChromeDriverDir = @"..\..\";
+        private static readonly string SiteBaseUri = @"http://localhost:64205/";
+
+        #region Default Config
+        [TestClass]
+        public class DefaultPollConfiguration : BasicPollTests
+        {
+            private static IVotingContext _context;
+
+            private const int truncatedTextLimit = 60;
+            private static Poll _defaultBasicPoll;
+
+            private IWebDriver _driver;
+
+            [TestInitialize]
+            public virtual void TestInitialise()
+            {
+                _driver = new NgWebDriver(new ChromeDriver(ChromeDriverDir));
+                _driver.Manage().Timeouts().SetScriptTimeout(TimeSpan.FromSeconds(10));
+                _driver.Manage().Timeouts().SetPageLoadTimeout(TimeSpan.FromSeconds(10));
+            }
+
+            [TestCleanup]
+            public void TestCleanUp()
+            {
+                _driver.Dispose();
+            }
+
+            [ClassInitialize]
+            public static void ClassInitialise(TestContext testContext)
+            {
+                _context = new TestVotingContext();
+
+                List<Option> testPollOptions = new List<Option>() {
+                new Option(){ Name = "Test Option 1", Description = "Test Description 1" },
+                new Option(){ Name = "Test Option 2", 
+                              Description = "A very long test description 2 that should exceed the character limit for descriptions" }};
+
+                // Open, Anonymous, No Option Adding, Shown Results
+                _defaultBasicPoll = new Poll()
+                {
+                    UUID = Guid.NewGuid(),
```

Comment:
Would it be clearer to use a fixed Guid for the test poll? If it doesn't get cleared up properly, then subsequent tests will fail to insert. Plus the addresses to be testing don't need to be generated on the fly, they can have a single point of change.